README.md

```
@@ -1,337 +1,67 @@
-# Kode Start by Kobe
+# Desafio Kobe Apps - Aplicativo Rick and Morty API 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)